docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -65,13 +65,13 @@ Currently, Codacy sends coverage data to your Git provider from both the old and
 
 ## If you are using the old Coverage status check to block merging pull requests on GitHub
 
-If you are using the old status check to block merging pull requests on GitHub, [edit the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch under the section **Require status checks to pass before merging**:
+If you are using the old status check to block merging pull requests on GitHub, [remove the old Codacy Coverage report as a required check by editing the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch under the section **Require status checks to pass before merging**:
 
 ![New Coverage status checks GitHub](../images/ala-695-update-status-checks-github.png)
 
-## Differences in coverage metrics between the old and new Coverage engines
+## Differences in coverage metrics between the old and new Coverage enginesx
 
-You may notice some differences in the coverage metrics reported by the old and new Coverage engines. This may happen because the new Coverage engine calculates coverage metrics considering all the files included in the coverage report, while the old Coverage engine ignores some files:
+You may notice some differences in the coverage metrics reported by the old and new Coverage engines. This may happen because the new Coverage engine calculates coverage metrics considering all the files included in the coverage report, while the old Coverage engine ignores some files:<!--NOTE Temporary comment for review purpose, to remove-->
```

Comment:
> Will customers need to adjust their quality gates for coverage?

They may have to, but we don't know how the numbers will change since each case will differ. The metrics may increase, decrease, or stay the same. The best we can do is mention that they may have to update their rules.

```suggestion
You may notice some differences in the coverage metrics reported by the old and new Coverage engines and may need to update your [coverage gate rules](../../repositories-configure/adjusting-quality-gates.md) accordingly. This may happen because the new Coverage engine calculates coverage metrics considering all the files included in the coverage report, while the old Coverage engine ignores some files:
```